FlowCrypt/Functionality/Services/Organisational Rules Service/OrganisationalRulesService.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  OrganisationalRulesService.swift
+//  FlowCrypt
+//
+//  Created by Yevhen Kyivskyi on 18.06.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+import Promises
+
+protocol OrganisationalRulesServiceType {
+    func fetchOrganisationalRulesForCurrentUser() -> Promise<OrganisationalRules>
+    func fetchOrganisationalRules(for email: String) -> Promise<OrganisationalRules>
```

Comment:
What is the difference between OrganizationalRules and ClientConfiguration (currently called DomainRules) in this code? Can we reuse the same object to avoid complexity?